common/src/test/java/com/revenuecat/purchases/common/offlineentitlements/OfflineCustomerInfoCalculatorTest.kt

```diff
@@ -154,30 +158,30 @@ class OfflineCustomerInfoCalculatorTest {
 
         // Made a purchase for p1m and then a purchase for not_bw after that one expired
         // You can't have an active purchase of the same product different base plans at the same time
-        val oneHourAgo = 1.hours.ago()
```

Comment:
We were making the same computation here as for request date making the expiration date to be sometimes after the request date (making the entitlement active), that's why the test was passing most of the times when we were checking for `isActive = true`. Although we actually wanted to check if `isActive = false` since it's an expired entitlement.